### DISCUSSION TITLE
Is it possible to customize generic OIDC IDP buttons?

### DISCUSSION THREAD

**Discussion by urandom** [zitadel/zitadel#5576] [Q&A]
When adding a generic OIDC IDP, the result looks like a transparent button on the login screen, with a slight border and the name of the IDP as the button label. Would it be possible to provide the background, foreground and icon for that button to better fit with the Google/Azure/Github/Gitlab buttons?

**Comment by fforootd** [comment]
We are working on an improved login experience that will allow you to take total control over the UI and the process.
After this (work started this week) that should be possible.

#4193
